### PR TITLE
[BUGFIX] Test property of the sidebar to set map position

### PR DESCRIPTION
**Pull request by xf-**

### CODE REVIEW

File: lib/sidebar.js [hunk 1]
```diff
@@ -21,12 +21,10 @@ define(function () {
...
 
     self.getWidth = function getWidth() {
-      if (sidebar.classList.contains('hidden')) {
+      if (sidebar.classList.contains('hidden') || sidebar.firstElementChild.offsetHeight === 0) {
         return 0;
       }
-
-      var small = window.matchMedia('(max-width: 630pt)');
-      return small.matches ? 0 : sidebar.offsetWidth;
+      return  sidebar.offsetWidth;
```
Comment (genofire) [inline lib/sidebar.js:27]: :+1:

File: lib/sidebar.js [hunk 2]
```diff
@@ -21,12 +21,10 @@ define(function () {
     sidebar.appendChild(container);
 
     self.getWidth = function getWidth() {
-      if (sidebar.classList.contains('hidden')) {
+      if (sidebar.classList.contains('hidden') || sidebar.firstElementChild.offsetHeight === 0) {
```
Comment (genofire) [inline lib/sidebar.js]: `sidebar.firstElementChild` wound be the `button` variable, firstElementChild is not really spezific ...

second, should it not be a `&&` ?

**Review by genofire — CHANGES_REQUESTED** [review]

File: lib/sidebar.js (hunk 2 above)
Comment (xf-) [reply lib/sidebar.js]: `||` is OK, if sidebar is closed (hidden) or is sidebar is on bottom 0 is correct value. If sidebar is visible we need the offset.

you are right about the `button`, the variable is available. But no more static width in code :).

File: lib/sidebar.js [hunk 3]
```diff
@@ -21,12 +21,10 @@ define(function () {
     sidebar.appendChild(container);
 
     self.getWidth = function getWidth() {
-      if (sidebar.classList.contains('hidden')) {
+      if (sidebar.classList.contains('hidden') || button.offsetHeight === 0) {
```
Comment (genofire) [inline lib/sidebar.js:24]: Ah top:
https://github.com/ffrgb/meshviewer/blob/develop/scss/modules/_sidebar.scss#L13

**Review by genofire — APPROVED** [review]

File: lib/sidebar.js (hunk 3 above)
Comment (xf-) [reply lib/sidebar.js:24]: this should fully handle the breakpoint and no code :)
https://github.com/ffrgb/meshviewer/blob/develop/scss/modules/_variables.scss#L24

**Pull request merged**